spark/src/main/scala/geotrellis/spark/timeseries/RDDTimeSeriesMethods.scala

```diff
@@ -0,0 +1,194 @@
+    left + right
+
+  def meanReduction(left: MeanResult, right: MeanResult): MeanResult =
+    left + right
+
+  def maxReduction(left: Double, right: Double): Double =
+    scala.math.max(left, right)
+
+  def minReduction(left: Double, right: Double): Double =
+    scala.math.min(left, right)
+
+  def sumReduction(left: Double, right: Double): Double =
+    left + right
+}
+
+abstract class RDDTimeSeriesMethods
+    extends MethodExtensions[TileLayerRDD[SpaceTimeKey]] {
+
+  def sumSeries(
+    polygon: MultiPolygon,
+    options: Options
+  ): Map[ZonedDateTime, Double] =
+    sumSeries(List(polygon), options)
+
+  def sumSeries(
+    polygon: MultiPolygon
+  ): Map[ZonedDateTime, Double] =
+    sumSeries(List(polygon), Options.DEFAULT)
+
+  def sumSeries(
+    polygons: Traversable[MultiPolygon]
+  ): Map[ZonedDateTime, Double] =
+    sumSeries(polygons, Options.DEFAULT)
+
+  def sumSeries(
+    polygons: Traversable[MultiPolygon],
+    options: Options
+  ): Map[ZonedDateTime, Double] = {
+    TimeSeries(
+      self,
+      SumDoubleSummary.handleFullTile,
+      RDDTimeSeriesFunctions.sumReduction,
+      polygons,
+      options
+    )
+      .collect()
+      .toMap
+  }
+
+  def minSeries(
+    polygon: MultiPolygon,
+    options: Options
+  ): Map[ZonedDateTime, Double] =
+    minSeries(List(polygon), options)
+
+  def minSeries(
+    polygon: MultiPolygon
+  ): Map[ZonedDateTime, Double] =
+    minSeries(List(polygon), Options.DEFAULT)
+
+  def minSeries(
+    polygons: Traversable[MultiPolygon]
+  ): Map[ZonedDateTime, Double] =
+    minSeries(polygons, Options.DEFAULT)
+
+  def minSeries(
+    polygons: Traversable[MultiPolygon],
+    options: Options
+  ): Map[ZonedDateTime, Double] = {
+    TimeSeries(
+      self,
+      MinDoubleSummary.handleFullTile,
+      RDDTimeSeriesFunctions.minReduction,
+      polygons,
+      options
+    )
+      .collect()
+      .toMap
+  }
+
+  def maxSeries(
+    polygon: MultiPolygon,
+    options: Options
+  ): Map[ZonedDateTime, Double] =
+    maxSeries(List(polygon), options)
+
+  def maxSeries(
+    polygon: MultiPolygon
+  ): Map[ZonedDateTime, Double] =
+    maxSeries(List(polygon), Options.DEFAULT)
+
+  def maxSeries(
+    polygons: Traversable[MultiPolygon]
+  ): Map[ZonedDateTime, Double] =
+    maxSeries(polygons, Options.DEFAULT)
+
+  def maxSeries(
+    polygons: Traversable[MultiPolygon],
+    options: Options
+  ): Map[ZonedDateTime, Double] = {
+    TimeSeries(
+      self,
+      MaxDoubleSummary.handleFullTile,
+      RDDTimeSeriesFunctions.maxReduction,
+      polygons,
+      options
+    )
+      .collect()
+      .toMap
+  }
+
+  def meanSeries(
+    polygon: MultiPolygon,
+    options: Options = Options.DEFAULT
+  ): Map[ZonedDateTime, Double] =
+    meanSeries(List(polygon), options)
+
+  def meanSeries(
+    polygon: MultiPolygon
+  ): Map[ZonedDateTime, Double] =
+    meanSeries(polygon, Options.DEFAULT)
+
+  def meanSeries(
+    polygons: Traversable[MultiPolygon]
```

Comment:
Sorry, I should have been more clear. The overloads that should exist are `Polygon` and `MultiPolygon`, not `MultiPolygon` and `Traversable[MultiPolygon]`.

If a user has a single polygon, which seems like a pretty regular case, then they can call the method with just the polygon.

If a user has a multipolygon, same thing.

If the user has a sequence of polygons or sequence of multipolygons, then they can wrap/union them into a multipolygon before calling this method.